jaworski_cellseg/pre_process_data_widget.py

```diff
@@ -62,12 +64,12 @@ def create_pre_process_data_widget(viewer: napari.Viewer, inferer_widget: Infere
         binary_map_threshold={"label": "Binary Threshold", "min": 0.0, "max": 100.0},
     )
     def pre_process_data_widget(
-        gaussian_checkbox: bool = True,
-        gaussian_factor: float = 1.0,
-        contrast_adjustment: bool = True,
-        contrast_lower: float = 5.0,
-        contrast_upper: float = 95.0,
-        binary_map_threshold: float = 90.0,
+        gaussian_checkbox: bool = config["gausianFilter"],
+        gaussian_factor: float = config["gaussian_sigma"],
+        contrast_adjustment: bool = config["contrast_adjustment"],
+        contrast_lower: float = config["contrast_min"],
+        contrast_upper: float = config["contrast_max"],
+        binary_map_threshold: float = config["binary_threshold"],
```

Comment:
Using mutable default values in functions can be problematic -- a change any of those values could change the function's behavior. How is this function to be used? Could you just pass the dict and unpack the data inside?

Reply:
Alternatively, you could organize the data in `config` in a `dataclass`. This is better than a dict because you can add type annotations to each of the fields.

Reply:
#10